pkg/engine/cosmosdb.go

```diff
@@ -31,16 +31,13 @@ func createCosmosDBAccount() map[string]interface{} {
 					"failoverPriority": 1,
 				},
 			},
+			"name":                             "[variables('cosmosAccountName')]",
+			"primaryClientCertificatePemBytes": "[variables('cosmosDBCertb64')]",
 		},
 		"tags": map[string]string{
 			"defaultExperience": "Etcd",
 		},
-		"consistencyPolicy": map[string]interface{}{
```

Comment:
This root "consistencyPolicy" object is superfluous, this is supposed to be a `properties` property.